spec/fixtures/emails/reply_to_different_to_from_quoted_display_name.eml

```diff
@@ -7,7 +7,7 @@ Message-ID: <3848c3m98r439c348mc349@test.mailinglist.com>
 Mime-Version: 1.0
 Content-Type: text/plain
 Content-Transfer-Encoding: 7bit
-X-Original-From: Marston, John <johnmarston@reddeadtest.com>
+X-Original-From: "Marston, John" <johnmarston@reddeadtest.com>
```

Comment:
Having a comma `,` in an address field means that there are more than one address.